docs/source/02-11-native-mobile.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Integrating on your native mobile application
+permalink: /v5/integrating/mobile/
+---
+
+Integration with native mobile applications is done through web view, you will need to host an HTML page which contains your embed code and open a web view to this page. There are many approaches to integrating native and web applications across different mobile operating systems, but any integration will involve the following steps:
+
+1. Host your embed code (find your **Embed code** under **Configure** > **Organization** > **Site Details**) in an HTML page served over HTTPS. For example, `https://yoursitename.com/coral.html`
+2. Add the domain to the list of permitted domains for your site under **Configure** > **Organization** > **Site Details**
+3. Create a web view in your native application which points to this URL
+4. Pass an SSO access token through to the embed code to log in your user. See [Single Sign On](/talk/v5/integrating/sso/) for information on how to generate an SSO token. There are multiple ways to pass data from a native application to a web view, one method is to encode the access token in a query parameter on the URL (ex. `https://yoursitename.com/coral.html?accessToken=ssoToken`) and retrieve the token from the embed code. You can then pass through the `accessToken` option passed to `createStreamEmbed`:
```

Comment:
I'd suggest making them include it in the hash instead:

```suggestion
4. Pass an SSO access token through to the embed code to log in your user. See [Single Sign On](/talk/v5/integrating/sso/) for information on how to generate an SSO token. There are multiple ways to pass data from a native application to a web view, one method is to encode the access token in a query parameter on the URL (ex. `https://yoursitename.com/coral.html#accessToken=ssoToken`) and retrieve the token from the embed code. You can then pass through the `accessToken` option passed to `createStreamEmbed`:
```

This ensure that the access token isn't logged via access logs.